target/linux/ramips/dts/mt7620a_netgear_ex6120.dts

```diff
@@ -0,0 +1,58 @@
+/* This file is released into the public domain */
+
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
```

Comment:
Consider renaming that file to mt7620a_netgear_ex3700_ex61xx.dtsi and update the name in the relevant DTS files.

Reply:
Not to be rude, but I do not understand the consequences of renaming a file in this project.
Rename considered but deemed too risky. I'm happy to open another issue to do this in a secondary step that can be blamed separately.

Reply:
The consequences of renaming a DTSI in target/linux/ramips/dts are that other files in the same directory will have to use the updated name for including it. So, just grep for the file name in that directory and replace the cases which show up.

The reason for this is that if someone wanted to change mt7620a_netgear_ex3700_ex6130.dtsi for some reason, he could rely on the file name and would possibly introduce a regression for ex6120. Therefore it seems much more desirable to adjust the file name now, so the names of all devices are covered, particularly since this has no downside.